Cpp_test/build_and_test.sh

```diff
@@ -4,29 +4,27 @@ rm -f testFile/ipc_send
 rm -f testFile/ipc_receive
 
 echo "Building the program"
-bazel build //main:ipc_send
-bazel build //main:ipc_receive
-
-mkdir build
-
-cp bazel-bin/main/ipc_send build
-cp bazel-bin/main/ipc_receive build
```

Comment:
Why did you deleted the copies the user is supposed to run? It would make more sense to run the tests with the copies in build-folder than to make the user use testFiles.